roadmaps/configuration.md

```diff
@@ -82,9 +86,6 @@ and configuration via
 [toolchains](https://docs.bazel.build/versions/master/toolchains.html)**
```

Comment:
@katre - is this June landing estimate at all reasonable for Java toolchain support?

Reply:
Forwarding question to @iirina, who is doing the work.